common/models/User-Identity.js

```diff
@@ -145,27 +147,42 @@ export default function(UserIdent) {
         user.picture = picture;
         userChanged = true;
       }
-
       if (!githubRegex.test(provider) && profile) {
         user[provider] = getUsernameFromProvider(provider, profile);
         userChanged = true;
+        if (userChanged) {
```

Comment:
Any reason that you put an `if` here? I'm asking because one line above you assign `true` to `userChanged` thus that condition will always be true. Am I wrong?